### DISCUSSION TITLE
Add city name to the business window

### DISCUSSION THREAD

**Discussion by hurzelchen** [voxeltycoon/issues#399] [UI&UX]
When showing multiple shops you can easily lose track as the UI doesn't show which shop is which

![image](https://user-images.githubusercontent.com/10977318/61246008-21ed5380-a74e-11e9-98b3-ddece768b983.png)

A simple fix would be adding the city's name to the tile of the window.

**Comment by victorleblais** [comment]
Here's how it could look like:
![image](https://user-images.githubusercontent.com/4365075/115540184-1f624b00-a29e-11eb-9fbd-b94733981ddb.png)
